turbo/jsonrpc/eth_receipts.go

```diff
@@ -509,7 +539,7 @@ func (api *APIImpl) GetBlockReceipts(ctx context.Context, numberOrHash rpc.Block
 	}
 	receipts, err := api.getReceipts(ctx, tx, block)
 	if err != nil {
-		return nil, fmt.Errorf("getReceipts error: %w", err)
+		return nil, errors.New(fmt.Sprintf("getReceipts error: %s", err))
```

Comment:
`%w` indicates error wrapping - you may not want to change that to `%s` usually since there might be a caller up the stack that uses `errors.Is`

Reply:
what's the reason for changing this?